src/peer_manager.rs

```diff
@@ -1148,6 +1150,13 @@ impl PeerManager {
 
     /// Marks the given peer as "connected and waiting for `NodeIdentify`".
     pub fn connected_to(&mut self, peer_id: &PeerId) {
+        // ConnectSuccess may received after established tunnel to peer (and inserted into RT)
```

Comment:
ConnectSuccess may be received after establishing a tunnel to peer (and adding to RT)